modules/nw-sriov-supported-devices.adoc

```diff
@@ -27,6 +37,21 @@
 |8086
 |158b
 
+|Intel
+|E810-CQDA2
```

Comment:
Please also add `E810-2CQDA2` which has the same Vendor and Device IDs with `E810-CQDA2`.

Reply:
In #35174, we consolidated variants of the same device when the vendor and device ID matched. Is there something different about `E810-2CQDA2` that means it should have a separate listing?

Reply:
Looking at the product pages, they are almost the same, except some minor differences in the bracket height and Data Rate Per Port.

E810-CQDA2: https://ark.intel.com/content/www/us/en/ark/products/192558/intel-ethernet-network-adapter-e810cqda2.html
E810-2CQDA2: https://ark.intel.com/content/www/us/en/ark/products/210969/intel-ethernet-network-adapter-e8102cqda2.html

Let me double check.

Reply:
also side by side comparison for reference:
https://ark.intel.com/content/www/us/en/ark/compare.html?productIds=210969,192558